webapp/components/Comment/Comment.vue

```diff
@@ -141,6 +151,10 @@ export default {
     },
   },
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-10T19:01:37Z" title="Friday, January 10th 2020, 8:01:37 pm +01:00">Jan 10, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -168,6 +176,23 @@ export default {
         this.$toast.error(err.message)
       }
     },
+    answerComment() {
+      const slug =
+        '<a class="mention" href="/profile/' +
+        this.comment.author.id +
+        '" data-mention-id="' +
+        this.comment.author.slug +
+        '" target="_blank" contenteditable="false">@d' +
+        this.comment.author.slug +
```


Use backticks for multi-line strings. Use string interpolation.

` <- thats a backtick
${} <- this is how you can do string interpolation

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-14T17:40:50Z" title="Tuesday, January 14th 2020, 6:40:50 pm +01:00">Jan 14, 2020</time>_

---


 
      this.editor.commands.mention({ id: message.id, label: message.slug })